homeassistant/components/devolo_home_control/climate.py

```diff
@@ -47,6 +48,22 @@ class DevoloClimateDeviceEntity(DevoloMultiLevelSwitchDeviceEntity, ClimateEntit
 
     @property
     def current_temperature(self) -> Optional[float]:
+        """Return the current temperature."""
+        if hasattr(self._device_instance, "multi_level_sensor_property"):
+            return next(
+                (
+                    multi_level_sensor.value
+                    for element_uid, multi_level_sensor in self._device_instance.multi_level_sensor_property.items()
+                    if multi_level_sensor.sensor_type == "temperature"
+                ),
+                self._value,
```

Comment:
To conform with the comment in line 63.
```suggestion
                self.target_temperature,
```